Applications/Utils/GeoTools/MoveGeometry.cpp

```diff
@@ -0,0 +1,85 @@
+/**
+ * \file MoveGeometry.cpp
+ * \author Karsten Rink
+ * \date   2015-08-04
+ * \brief  A small tool to translate geometries.
+ *
+ * \copyright
+ * Copyright (c) 2012-2015, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ */
+
+// ThirdParty
+#include "tclap/CmdLine.h"
+
+// ThirdParty/logog
+#include "logog/include/logog.hpp"
+
+// BaseLib
+#include "LogogSimpleFormatter.h"
+
+// MathLib
+#include "MathLib/Vector3.h"
+
+// FileIO
+#include "FileIO/XmlIO/Qt/XmlGmlInterface.h"
+
+// GeoLib
+#include "GeoLib/GEOObjects.h"
+
+#include <QtGui/QApplication>
+
+int main(int argc, char *argv[])
+{
+	QApplication app(argc, argv);
+	LOGOG_INITIALIZE();
+	logog::Cout *logogCout(new logog::Cout);
+	BaseLib::LogogSimpleFormatter *custom_format (new BaseLib::LogogSimpleFormatter);
+	logogCout->SetFormatter(*custom_format);
+
+	TCLAP::CmdLine cmd("Moves the points of a geometry by a given displacement vector.", ' ', "0.1");
+	TCLAP::ValueArg<double> z_arg("z", "z", "displacement in z direction", false, 0.0, "z-displacement");
+	cmd.add(z_arg);
+	TCLAP::ValueArg<double> y_arg("y", "y", "displacement in y direction", false, 0.0, "y-displacement");
+	cmd.add(y_arg);
+	TCLAP::ValueArg<double> x_arg("x", "x", "displacement in x direction", false, 0.0, "x-displacement");
+	cmd.add(x_arg);
+	TCLAP::ValueArg<std::string> geo_output_arg("o","output", "output geometry file (*.gml)", true, "", "output file");
```

Comment:
Long lines are difficult to read. http://ufz.github.io/styleguide/cppguide.xml#Line_Length